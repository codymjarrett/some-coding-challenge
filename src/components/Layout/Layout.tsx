import React from 'react'

import Header from '../Header/Header'

interface ChildrenProps {
	children: any
}

const Layout: React.FC<ChildrenProps> = ({ children }) => (
	<div className="layout">
		<Header />
		{children}
	</div>
)

export default Layout
