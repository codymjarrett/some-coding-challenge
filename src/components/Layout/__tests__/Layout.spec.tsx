import React from 'react'
import { render, cleanup } from '@testing-library/react'
import { BrowserRouter as Router } from 'react-router-dom'

import Layout from '../Layout'

afterEach(cleanup)

test('<Layout />', () => {
	const { container, debug } = render(
		<Router>
			<Layout />
		</Router>
	)
	expect(container.firstChild).toMatchSnapshot()
})
