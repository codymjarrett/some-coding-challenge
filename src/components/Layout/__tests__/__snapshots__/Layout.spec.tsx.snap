// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> 1`] = `
<div
  class="layout"
>
  <header
    class="header"
  >
    <div
      class="header__wrapper"
    >
      <h1
        class="header__title"
      >
        <a
          class="header__link"
          href="/"
        >
          San Fran Movie Fun
        </a>
      </h1>
    </div>
  </header>
</div>
`;
